lib/internal/process/nextTick.js

```diff
@@ -0,0 +1,228 @@
+'use strict';
+
+exports.setup = setupNextTick;
+
+function setupNextTick(NativeModule) {
```

Comment:
Why do you need `NativeModule`? I think you can just use `require` at this point